pkg/events/marshalers.go

```diff
@@ -30,6 +30,9 @@ func (s *HTTPRequest) MarshalLog(addField func(key string, value interface{})) {
 	addField("http.url_details.path", s.Path)
 	addField("http.url_details.uri", s.URI)
 	addField("http.url_details.endpoint", s.Endpoint)
+	if s.Route != "" {
+		addField("http.route", s.Route)
+	}
```

Comment:
issue: as it turns out, this change is in an autogenerated file, so this change gets reverted every time `make gogenerate` is run, which also happens every time a re-stencil happens.